draft-packages/tooltip/KaizenDraft/Tooltip/utils/isSemanticElement.spec.tsx

```diff
@@ -0,0 +1,68 @@
+import React from "react"
+import { Button, IconButton } from "@kaizen/button"
+import { Icon } from "@kaizen/component-library"
+import ArrowIcon from "@kaizen/component-library/icons/arrow-right.icon.svg"
+import { isSemanticElement } from "./isSemanticElement"
+
+describe("isSemanticElement", () => {
+  it("will return true if provided a native element with a semantic role", () => {
```

Comment:
Just to distinguish this from the case of adding a role to a non-semantic element
```suggestion
  it("returns true if provided a semantic element", () => {
```